Presentation/WindowsClient/Models/TaskRunners/ApsimXporter.cs

```diff
@@ -181,7 +181,6 @@ private async Task<IModel> CreateExperiment(string name, int id)
                             b_interception = 1,
                             c_interception = 0,
                             d_interception = 0,
-                            soil_albedo = 0.13,
```

Comment:
what does this have to do with importing?

Reply:
It doesn't, fixing this was a side-effect of resolving the earlier merge conflict.